README.rst

```diff
@@ -47,24 +47,32 @@ Here's an example
 .. code-block:: python
 
     # mycode.py
+    import procrastinate
 
     # Make an app in your code
-    app = procrastinate.App(connector=procrastinate.AiopgConnector())
+    app = procrastinate.App(connector=procrastinate.AiopgConnector(host="localhost", user="postgres", password="password"))
 
     # Open the connection to the database
-    app.open()
+    with app.open():
 
-    # Then define tasks
-    @app.task(queue="sums")
-    def sum(a, b):
-        with open("myfile", "w") as f:
-            f.write(str(a + b))
+        # Automatically apply the `procrastinate` schema to the database
+        try:
+            app.schema_manager.apply_schema()
+        except procrastinate.exceptions.ConnectorException:
+            # Schema has already been applied
+            pass
 
-    # Launch a job
-    sum.defer(a=3, b=5)
+        # Then define tasks
+        @app.task(queue="sums")
+        def sum(a, b):
+            with open("myfile", "w") as f:
+                f.write(str(a + b))
+
+        # Launch a job
+        sum.defer(a=3, b=5)
 
-    # Somewhere in your program, run a worker
-    app.run_worker(queues=["sums"])
+        # Somewhere in your program, run a worker
```

Comment:
I'd be tempted to change this comment to this one:

```
# Somewhere in your program (actually, often a different program than the one launching jobs), run a worker
```

Reply:
Added 👍